### PR TITLE
Add missing build ids and rename *.txt channels

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

**Comment by drpatelh** [comment]
Stringtie tests are failing because we don't currently have any tests set-up for it.

**Comment by drpatelh** [comment]
@JoseEspinosa be great if you can have a quick look and approve here please :) I want to re-install some of these modules in viralrecon. Ignore the failing tests for Stringtie, those can be added in separate PR.

File: software/bcftools/stats/main.nf [hunk 1]
```diff
@@ -21,8 +21,8 @@ process BCFTOOLS_STATS {
     tuple val(meta), path(vcf)
 
     output:
-    tuple val(meta), path("*.txt"), emit: stats
-    path  "*.version.txt"         , emit: version
+    tuple val(meta), path("*stats.txt"), emit: stats
+    path  "*.version.txt"              , emit: version
```
Comment (JoseEspinosa) [inline software/bcftools/stats/main.nf:25]: Change the meta.yml file accordingly

**Review by JoseEspinosa — APPROVED** [review]
LGTM!
Just a small comment to add to the PR and ready to go! 🚀

File: software/bcftools/stats/main.nf (hunk 1 above)
Comment (drpatelh) [reply software/bcftools/stats/main.nf:25]: Not sure I understand?

Comment (drpatelh) [reply software/bcftools/stats/main.nf:25]: https://github.com/nf-core/modules/pull/164/commits/d2508e3c0308ff601aa30b5f21d3a2e1eac85d75

Comment (JoseEspinosa) [reply software/bcftools/stats/main.nf:25]: My mistake, my comment was wrong.

**Comment by drpatelh** [comment]
Thank you!

**Pull request merged**